<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/dog.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionConnect_CAN"/>
    <addaction name="actionDisconnect_CAN"/>
    <addaction name="actionTerminal"/>
    <addaction name="actionReceive_Error"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionJoint"/>
    <addaction name="actionJoint_Speed_Meas"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionSingle_Joint_Control"/>
    <addaction name="actionJoint_Control"/>
    <addaction name="actionOffline_Control"/>
    <addaction name="actionRemote_Control"/>
    <addaction name="actionSequence_Control"/>
    <addaction name="actionArm_Control"/>
    <addaction name="actionReconfig_Control"/>
   </widget>
   <addaction name="menuTool"/>
   <addaction name="menuPlot"/>
   <addaction name="menuControl"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConnect_CAN">
   <property name="text">
    <string>Connect CAN</string>
   </property>
  </action>
  <action name="actionJoint">
   <property name="text">
    <string>Joint</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionJoint_Control">
   <property name="text">
    <string>Joint Control</string>
   </property>
  </action>
  <action name="actionJoint_Speed_Meas">
   <property name="text">
    <string>Joint Speed Meas</string>
   </property>
  </action>
  <action name="actionOffline_Control">
   <property name="text">
    <string>Offline Control</string>
   </property>
  </action>
  <action name="actionSingle_Joint_Control">
   <property name="text">
    <string>Single Joint Control</string>
   </property>
  </action>
  <action name="actionReceive_Error">
   <property name="text">
    <string>Receive Error</string>
   </property>
  </action>
  <action name="actionRemote_Control">
   <property name="text">
    <string>Remote Control</string>
   </property>
  </action>
  <action name="actionDisconnect_CAN">
   <property name="text">
    <string>Disconnect CAN</string>
   </property>
  </action>
  <action name="actionSequence_Control">
   <property name="text">
    <string>Sequence Control</string>
   </property>
  </action>
  <action name="actionArm_Control">
   <property name="text">
    <string>Arm Control</string>
   </property>
  </action>
  <action name="actionReconfig_Control">
   <property name="text">
    <string>Reconfig Control</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
